package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ivory",
-  "version": "0.10.1",
+  "version": "0.10.2",
```

Comment:
This should be a 'minor' increment, so 0.11.0, as this is new functionality rather than a bug fix or rebuild.

Reply:
👍🏼